asab/storage/elasticsearch.py

```diff
@@ -83,6 +114,42 @@ def session(self):
 		return self._ClientSession
```

Comment:
`auth=self._auth` will be always None with api_key... Do we want to update this given that we are adding api_key as a second authorization method?